apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/web/controllers/block.ex

```diff
@@ -20,14 +20,11 @@ defmodule OMG.ChildChainRPC.Web.Controller.Block do
   use OMG.ChildChainRPC.Web, :controller
 
   alias OMG.ChildChain
-  alias OMG.ChildChainRPC.Web.View
 
   def get_block(conn, params) do
     with {:ok, hash} <- expect(params, "hash", :hash),
          {:ok, block} <- ChildChain.get_block(hash) do
-      conn
```

Comment:
I'm not saying this is better or worse, but at least now it's consistent with watcher's controllers.